modules/data.remote/R/call_MODIS.R

```diff
@@ -111,11 +164,33 @@ call_MODIS <- function(outfolder = ".", start_date, end_date, lat, lon, size = 0
     output$lat <- round(output$lat, 4)
     output$lon <- round(output$lon, 4)
     
-    fname <- paste(product, "_", band, "output_", start_date, "_", end_date, "_", lat, "_", lon, ".csv", sep = "")
-    fname <- paste0(outfolder, "/", fname)
-    write.csv(output, fname)
-    return(output)}
-
+    if (QC_filter == T)
```

Comment:
It's already boolean -- no need to test for equality with `T`.
```suggestion
    if (QC_filter)
```

If you want to ensure it's a boolean and nothing else, you can use `if(isTRUE(QC_filter))`. Case in point:

``` r
1 == TRUE
#> [1] TRUE
isTRUE(1)
#> [1] FALSE
```

Reply:
please update @bailsofhay